src/address.js

```diff
@@ -254,10 +285,11 @@ Returns null if no address is found.
   /**
    * Retrieves the latest account and token balances for the specified address.
    *
-   * @param hash - The address of the account.
-   * @param filterOptions - The filter options associated with the request.
-   * @returns The balance data of the account.
+   * @param {string} hash - The address of the account.
+   * @param {object} [filterOptions] - The filter options associated with the request. See [docs](https://docs.amberdata.io/reference#account-balances) for more details.
+   * @returns {Promise<object>} The balance data of the account.
    * @example
+   * const balances = await web3data.address.getBalances('0x06012c8cf97bead5deae237070f9587f8e7a266d')
```

Comment:
@taylorjdawson another weird thing here... Why do we need this method? this is very confusing